### ISSUE TITLE
Keybase Install Error on startup, Keybase Filesystem not working

### ISSUE THREAD

**Issue by Edeholland** [keybase/client#20234]
This morning I upgraded my MacBook Pro to Catalina. When restarting, I got a popup with the following message:

```Keybase Install Error

We were unable to load KBFS (Fuse kext). This may be due to a limitation in MacOS where there aren't any device slots available. Device slots can be taken up by apps such as VMWare, VirtualBox, anti-virus programs, VPN programs and Intel HAXM.```

my log id: bb60401a3901223e8c72421c

When I launch Keybase and enable Keybase in Finder, it keeps on loading and nothing is happening. I would like to get the Keybase filesystem to work again.

**Comment by Edeholland** [comment]
![image](https://user-images.githubusercontent.com/34871543/66476159-33b46d80-ea95-11e9-9112-29303e6a904e.png)
I now get an error while it's trying to enable Keybase in Finder.

**Comment by strib** [comment]
@Edeholland please send more logs for the newest error you reported, and we can take a look.

**Comment by itzsaga** [comment]
Yeah, the upgrade did something and it placed a "Relocated Items" folder on the desktop. `keybase` is in that folder and here is the info from Apple.
![Screen Shot 2019-10-09 at 11 17 19](https://user-images.githubusercontent.com/11750061/66499963-6014bd80-ea86-11e9-81a5-df024c7bff0f.png)
I'll be happy to send any additional info, just direct me to how to get it for you please.

**Comment by itzsaga** [comment]
Digging into it all my files **are** available in my Keybase folder (as seen on the Desktop in the screenshot) but I received the same initial error as @Edeholland. It appears that the app is functioning properly. Not sure if there are any underlying things that might be messed up.

**Comment by Edeholland** [comment]
> @Edeholland please send more logs for the newest error you reported, and we can take a look.

I just send more logs with the following ID: d08c28ad7a13f6aaa06c111c

@itzsaga My files are available in the application, but the filesystem is not working. It also seems my files used to be placed in /Keybase but are now placed in /Volume/Keybase. Could this be part of the issue?

**Comment by strib** [comment]
Yes, with Catalina, macOS doesn't let us use `/keybase` anymore, so we mount the redirector at `/Volumes/Keybase` (which redirects you to the local user's mount), and the local user continues to mount their file system at `/Keybase/Volumes (LOCAL_USER_NAME)`.

Are your files accessible via `/Volumes/Keybase`, or is that directory just empty now?  If it's empty, maybe try disabling Finder integration (in the Settings menu) and then re-enabling it?  Not sure why it's not working for you...

**Comment by llimllib** [comment]
I was able to find my files in `/Volumes/Keybase` after upgrade, but it took me quite a while to figure out where they were

**Comment by Edeholland** [comment]
@strib I was not able to find my files under `/Volumes/Keybase`. I only had the `/Keybase/Volumes (LOCAL_USER_NAME)` folder. I reinstalled Keybase and now I have both directories. 

I still have to change all my mounts from `/Keybase` to `/Volumes/Keybase`. It would be good to see this documented somewhere so I could have prepared for it.

**Comment by kmbremner** [comment]
I've just had this error too on Catalina.
Logs sent: 5bb81a2a7331dd5e85b3341c
